### ISSUE TITLE
Why do we need to "click to start" playing the game

### ISSUE THREAD

**Issue by grymmy** [hackclub/sinerider#400]
### Description

Is there a technical or product reason to require the user to "click to start" when navigating to the sinerider page?  It seems like an unnecessary friction point.

**Comment by polytroper** [comment]
Yes, it is required by browsers to initiate audio for the page. Without this step, the audio in the loading screen will fail to play.

**Comment by polytroper** [comment]
https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide

**Comment by camdan-me** [comment]
You should make the whole screen clickable. Also, there's gonna be more info on it right?

**Comment by karamassie** [comment]
@polytroper what's your call here?

**Comment by polytroper** [comment]
@camdan-me the whole screen is clickable

Closing
